Include/cpython/pytime.h

```diff
@@ -2,8 +2,7 @@
 #ifndef Py_PYTIME_H
 #define Py_PYTIME_H
 
-#include "pyconfig.h" /* include for defines */
-#include "object.h"
+#include "../object.h"
```

Comment:
I don't think that people should include pytime.h directly, and so I suggest to simply remove these includes. Let people respect the Python C API documentation which suggests to **only** include "Python.h".